bin/install

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -Eiuo pipefail
```

Comment:
I'm getting the following error in Mac OS X for both this script and the `download` one:
```
/Users/juan.facorro/.asdf/plugins/clojerl/bin/install: line 3: set: -i: invalid option
```

Reply:
Yes, it's a typo, must be an `e` . I'll provide a fix asap

Reply:
@jfacorro done: https://github.com/clojerl/asdf-clojerl/pull/4

Reply:
Cool! Thank you 😄